### PR TITLE
New schema 2404 11

### PR DESCRIPTION
**Pull request by pietrodelugas**
add two test schemas for updarting cpv_pw schema  

   
    * add option for wannier type atomic hubbard projection type
    * add esm informations in outputPBC element (needed to fix
      post-processing issues)
    * add input element for the setting twochem implementation 
    * add element for in `output/band_structure` for recovering data
      for using twochem implementation with linear response.
    * add more options in the lowhighType restriction to allow all available
      options for disck_io and verbosity options.

### CODE REVIEW

**Comment by pietrodelugas** [comment]
tag @giomarini1993 for the two_chem pot things

File: PW_CPV/previous_schemas/qes_240411.xsd [hunk 1]
```diff
@@ -0,0 +1,1391 @@
...
+    <sequence>
+      <choice minOccurs="1" maxOccurs="1">
+         <element type="qes:atomic_positionsType" name="atomic_positions" minOccurs="0"  />
+         <element type="qes:wyckoff_positionsType" name="wyckoff_positions" minOccurs="0" />
+         <element type="qes:atomic_positionsType" name="crystal_positions" minOccurs="0" />
+      </choice>
+      <element type="qes:cellType" name="cell"/>
+    </sequence>
+    <attribute type="positiveInteger" name="nat" />
+    <attribute type="integer" name = "num_of_atomic_wfc" use="optional"/>
```
Comment (mbercx) [inline PW_CPV/previous_schemas/qes_240411.xsd:241]: This one used to be in the `band_structure`, I believe. Why was this moved to the `atomic_structure`? 🤔

Comment (pietrodelugas) [reply PW_CPV/previous_schemas/qes_240411.xsd:241]: We initially needed to read `num_of_atomic_wfc` before allocating the data structures for reading band_structure. Instead of moving it, we could put it in both places.
